README.md

```diff
@@ -70,14 +81,42 @@ class _HomeState extends State<Home> {
   @override
   void initState() {
     super.initState();
+    checkVersion();
+  }
 
-    Future.delayed(Duration.zero, () {
-      NativeUpdater.displayUpdateAlert(
-        context: context,
-        requiresUpdate: true,
-        appStoreUrl: '<Your App Store URL>',
-      );
-    });
+  Future<void> checkVersion() async {
+    /// For example:
+    /// You've got status code of 412 from the response of HTTP request.
+    /// Let's say the statusCode 412 is requires you to force update
+    /// and statusCode 200 ik OK but new version available.
+
+    int statusCode = 412; /// Try by switching the value to 412 or 200
+
+    if (statusCode == 412) {
+      Future.delayed(Duration.zero, () {
+        NativeUpdater.displayUpdateAlert(
+          context: context,
+          forceUpdate: true,
+          // appStoreUrl: '<Your App Store URL>',
+          // titlePrefix: 'Perbaharui',
+          // description: '<Your description>',
+          // updateButtonLabel: 'Perbaharui',
+          // closeButtonLabel: 'Tutup',
+        );
+      });
+    } else if (statusCode == 200) {
```

Comment:
@JeffryHermanto, this could be a misleading example because IMO if we want to provide an optional update, we should compare between the local version and the newest version that we receive from the API. WDYT?

Reply:
Can you provide your example version?

Reply:
I'd probably go with this
```suggestion
   int newVersion = 2 //this could get from the API
   int currentVersion = 1 //this could be kept in our local
   ...
    } else if (currentVersion < newVersion) {
    //Show optional updates
    }
```
wdyt?

Reply:
Resolved, it's on #2